mech-lore.lic

```diff
@@ -29,6 +29,15 @@ class MechLore
     fput "wear my #{@settings.hand_armor}"
   end
 
+  def dump_junk
```

Comment:
What would you think about moving this into `common`?

Maybe something like:

```
 def dump_junk?(dump_junk, junk_timer, dump_item_count)
    return unless dump_junk
    return unless Time.now - junk_timer > 300
    return unless DRRoom.room_objs.size > dump_item_count

    fput('dump junk')
  end
```

And then the usage would be:

```
def blah
  ...
  @junk_timer = Time.now if dump_junk?(@settings.dump_junk, @junk_timer, @settings.dump_item_count)
  ...
end
```

Is that too verbose?